src/java.desktop/macosx/classes/sun/lwawt/macosx/CPrinterJob.java

```
@@ -371,6 +371,8 @@ public SecondaryLoop run() {
 
                     try {
                         printLoop(true, firstPage, lastPage);
+                    } catch (PrinterAbortException pex) {
+                        throw new PrinterAbortException(pex.getMessage());
```

Why not re-throw the original exception ?

---

Did you miss this question ?

---

oh NM, this comments view doesn't show that it was fixed.